recipe/patches/0002-unvendor-pyproject-metadata.patch

```diff
@@ -0,0 +1,67 @@
+From 1fdffa9ea898e333bd8216ba5c3f9581948c38ed Mon Sep 17 00:00:00 2001
+From: Udo-Peter Steyer <ous8292@gmail.com>
+Date: Thu, 8 Aug 2024 10:51:02 -0500
+Subject: [PATCH] unvendor pyproject-metadata
```

Comment:
What's the reason for unvendoring? It was initially vendored for a reason: https://github.com/scikit-build/scikit-build-core/pull/703.

Reply:
Removed the unvendoring patch!